suncalc.go

```diff
@@ -168,11 +184,14 @@ func getSetJ(h float64, lw float64, phi float64, dec float64, n float64, M float
 	return solarTransitJ(a, M, L)
 }
 
-// calculates sun times for a given date and latitude/longitude
-func GetTimes(date time.Time, lat float64, lng float64) map[DayTimeName]DayTime {
+// calculates sun times for a given date and latitude/longitude, and,
+// the observer height (in meters) relative to the horizon, you can set it to 0 if unknown
+func GetTimes(date time.Time, lat float64, lng float64, height float64, location *time.Location) map[DayTimeName]DayTime {
```

Comment:
Could you make the latest parameter vararg? Something like [this](https://stackoverflow.com/a/71099156) 
This way it would be optional and the change wouldn’t be a breaking one?

Reply:
I hesitate to make a vararg as it can lead to unintended results of extra unused variables being sent to a function.  I prefer passing a structure with defined and undefined values.